package-build/tools/vcpkgInstall.ps1

```diff
@@ -5,33 +5,66 @@ $RootDir = 'C:\opt'
 $InstallDir = "$RootDir\vcpkg"
 $VcpkgVersion = "2020.04"
 
-try {
+try
+{
   New-Item -ItemType Directory -Path $RootDir -Force | Out-Null
 
+  # shadow the Vcpkg environment variables which could break the deployment.
+  if (Test-Path env:VCPKG_ROOT)
+  {
+    Write-Warning "%VCPKG_ROOT% is defined. Removing from current session."
```

Comment:
Removing the vcpkg library manager in order to reinstall it.

Reply:
I will reduce this one to an normal log. No action is required from users.